src/components/NavbarContainer.jsx

```diff
@@ -23,6 +23,12 @@ class NavbarContainer extends Component {
     this.setState({ showSearch: !showSearch });
   }
 
+  onLogoutClick = (e) => {
+    e.preventDefault();
+    localStorage.clear();
+    window.location.href = '/login';
```

Comment:
I believe we have used react router for this application. This would have been ideal if this were a vanilla-Js application. I also believe we could have ensured that since this is an action we expect to be executed, then this implementation should have been done in your `actions` not here.

Reply:
Alright, thanks